.travis.yml

```diff
@@ -79,7 +79,6 @@ install:
   - cd ../..
 
 script:
-  - pycodestyle --show-source --show-pep8 $PYCODESTYLE_CONVENTIONS $CGAL_PYTHON_EXAMPLES || pycodestyle --statistics $PYCODESTYLE_CONVENTIONS -qq $CGAL_PYTHON_EXAMPLES
```

Comment:
I disagree with this change. The PEP8 document is recognized as a standard that defines a uniform coding style for all Python code. If the style checker reports error, then fix them: install `pycodestyle` locally, then use it and fix all the errors, instead of removing the check.